src/elementary/IsotopeID/src/register.hpp

```diff
@@ -0,0 +1,2 @@
+// register the regex
+const std::regex IsotopeID::regex{ "([A-Za-z][a-z]?)(\\d{1,3})" };
```

Comment:
the regex is a static member so it needs to be initialised outside of the class. I can't make it constexpr either since it is not a literal type. Please note that the register include is outside the class definition.

Reply:
Oh this is one of those C++ things that always bites me.